src/installers/helm.ts

```diff
@@ -33,38 +33,13 @@ export class HelmHelper {
         enabled: () => {
           return flags.tls
         },
-        task: async (ctx: any, task: any) => {
-          const kh = new KubeHelper()
-          const exists = await kh.secretExist('che-tls')
-          if (!exists) {
-            throw new Error('TLS option is enabled but che-tls secret does not exist in default namespace. Example on how to create the secret: kubectl create secret generic che-tls --from-literal=ACME_EMAIL=my@email-address.com')
-          }
-          const tlsEmail = await kh.getSecret('che-tls')
-          if (tlsEmail === undefined) {
-            throw new Error('TLS option is enabled and che-tls secret is defined but there is no ACME_EMAIL field on this secret. Example on how to create the secret: kubectl create secret generic che-tls --from-literal=ACME_EMAIL=my@email-address.com')
-          }
-          ctx.tlsEmail = tlsEmail
-          task.title = `${task.title}...che-tls secret found.`
-        }
-      },
-      {
-        title: 'Check for cert-manager',
-        // Check only if TLS is enabled
-        enabled: () => {
-          return flags.tls
-        },
         task: async (_ctx: any, task: any) => {
           const kh = new KubeHelper()
-          const exists = await kh.apiVersionExist('certmanager.k8s.io')
+          const exists = await kh.secretExist('che-tls', `${flags.chenamespace}`)
           if (!exists) {
-            throw new Error(`TLS option is enabled but cert-manager API has not been found. Cert Manager is probably not installed. Example on how to install it:
-            $ kubectl create namespace cert-manager
-            $ kubectl label namespace cert-manager certmanager.k8s.io/disable-validation=true
-            $ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.8.1/cert-manager.yaml --validate=false
-
-            Please install cert-manager.`)
+            throw new Error(`TLS option is enabled but che-tls secret does not exist in '${flags.chenamespace}' namespace. Example on how to create the secret with TLS: kubectl create secret tls che-tls --namespace=che --key=privkey.pem --cert=fullchain.pem`)
```

Comment:
In the example command to create the secret you can replace `--namespace=che` with `--namespace=${flags.chenamespace}` so that the example uses the right namespace.

Reply:
fixed